test/schema_diff.sh

```diff
@@ -0,0 +1,2 @@
+diff=`git diff`
+grep -q 'schema.json' <<< $diff && echo "changes" exit 1 || echo "no changes" exit 0
```

Comment:
If we're going to use a shell script, we should specify the executable and (if bash) should pipefail: https://sipb.mit.edu/doc/safe-shell/